zhaquirks/xiaomi/__init__.py

```diff
@@ -414,6 +419,16 @@ class MotionCluster(LocalDataCluster, MotionOnEvent):
     reset_s: int = 70
 
 
+class DeviceTemperatureCluster(LocalDataCluster, DeviceTemperature):
+    """Device Temperature Cluster."""
+
+    cluster_id = DeviceTemperature.cluster_id
+
+    def update_attribute(self, attrid, value):
```

Comment:
This should already be in the latest zigpy, so no need to redefine it here.

Reply:
Without `def update_attribute(self, attrid, value):` it wasn't working on my "testbench" HA 2012.12.07

Reply:
You were right, sorry